packages/tizen_notification/lib/notification.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/services.dart';
+import 'package:tizen_app_control/app_control.dart';
+
+import 'src/types.dart';
+
+export 'package:tizen_app_control/app_control.dart';
+
+export 'src/enums.dart';
+export 'src/types.dart';
+
+// ignore: public_member_api_docs
+class TizenNotificationDetails {
+  /// [TizenNotificationDetails] contains detail information about the notification
+  /// being sent.
+  ///
+  /// [images] only supports type [List<NotificationImage>] and [NotificationImage].
+  const TizenNotificationDetails({
+    this.images,
+    this.properties,
+    this.appControl,
+    this.vibration,
+    this.sound,
+    this.displayApplist = DisplayApplist.all,
+    this.onGoing = false,
+  });
+
+  /// A single or a list of [NotificationImage].
+  ///
+  /// [NotificationImage] refers to any image shown on the notification panel.
+  final dynamic images;
```

Comment:
Can't you just use a List type?

Reply:
I've allowed a single NotificationImage because I believe many users will use not provide a list of NotificationImage in many cases. I thought its better for the users to use [NotificationImage] but I'll change it if it's not dart-like.